pandas/core/dtypes/cast.py

```diff
@@ -1012,12 +1012,10 @@ def convert_dtypes(
         object array contains pd.NA.
     dtype_backend : {'numpy_nullable', 'pyarrow'}, default 'numpy_nullable'
         Back-end data type applied to the resultant :class:`DataFrame`
-        (still experimental). Behaviour is as follows:
+        (still experimental). If specified, the behavior is as follows:
```

Comment:
This is also the behavior when not specified (the default being `"numpy_nullable"`). In other words, I think `If specified` is unnecessary.